.bazelci/presubmit.yml

```diff
@@ -205,15 +205,15 @@ tasks:
   ubuntu1804:
     name: "Min Bazel Version"
     # If updating the minimum bazel version, please also update /docs/index.md
-    bazel: "5.2.0"
+    bazel: "6.0.0"
```

Comment:
What necessitates the minimum bazel bump? Something in proto toolchains?

Reply:
Yeah, the proto_common dependency is not defined before bazel 6.0 when importing it from

```starlark
load("@rules_proto//proto:defs.bzl", "ProtoInfo", "proto_common")
```

https://github.com/bazelbuild/rules_proto/blob/master/proto/private/native.bzl#L28